mcfly.bash

```diff
@@ -21,8 +21,7 @@ MCFLY_SESSION_ID="$(command dd if=/dev/urandom bs=256 count=1 2> /dev/null | LC_
 export MCFLY_SESSION_ID
 
 # Find the binary
-MCFLY_PATH=${MCFLY_PATH:-$(which mcfly)}
```

Comment:
There was definitely a reason for $MCFLY_PATH to exist, but I can't recall anymore what it was.

Reply:
Just leave it out then? I can't find anything that reads it, just things setting it in the dev.* scripts.